### PR TITLE
Extending Docker image to have Bex screenshot extension for Behat

### PR DESCRIPTION
**Pull request by toni-kolev**
Adding `bex/behat-screenshot` to composer.

### CODE REVIEW

**Comment by lmakarov** [comment]
@toni-kolev can you please also provide an example of how you are using `bex/behat-screenshot`?

**Comment by toni-kolev** [comment]
@lmakarov I have added example configuration in the `behat.yml` and updated the `README` with some information about the extension. Let me know if there's anything else that is needed.

File: example/behat.yml [hunk 1]
```diff
@@ -48,6 +48,12 @@ default:
       print_outp: true
       loop_break: true
 
+     # Bex Screenshot
+    Bex\Behat\ScreenshotExtension:
+      image_drivers:
+        local:
+          screenshot_directory: "%paths.base%/<change_screenshots_folder_name>/"
```
Comment (lmakarov) [inline example/behat.yml]: Can we assume a default folder name, which works out of the box (is created automatically)?

Comment (toni-kolev) [reply example/behat.yml]: If you don't provide a folder screenshots are stored in `/tmp/behat-screenshot/`

image

If you provide a folder path and the folder does not exist, it is created.

Comment (lmakarov) [reply example/behat.yml]: If it's just set to `%paths.base%/screenshots/` by default, where will the screenshots go?

Comment (toni-kolev) [reply example/behat.yml]: `%paths.base%` refers to the base of Behat. This is the folder which contains the `behat.yml` and the `features` folder. In other words the `screenshots` folder will be sibling of the `features` folder.

image

Comment (lmakarov) [reply example/behat.yml]: > In other words the screenshots folder will be sibling of the `features` folder.

That sounds like a fine place to place the `screenshots` folder, so let's just go with that. This way users won't have to edit anything out of the box.

Comment (toni-kolev) [reply example/behat.yml]: I agree. Changes to `behat.yml` are commited.

**Review by lmakarov — APPROVED** [review]

**Comment by lmakarov** [comment]
Thanks!

**Pull request merged**